modules/dicom_archive/help/dicom_archive.md

```diff
@@ -1,17 +1,9 @@
 # Dicom Archive
 
-The Dicom Archive module displays all scans that have been uploaded
-to LORIS, whether or not they have been inserted or not (due to protocol
-violations, mislabeling, not having been run, etc.) It's a filterable archive
-of every single dicom that this LORIS instance knows about.
+This module displays all scans that have been uploaded to LORIS. This includes scans that have been inserted, as well as those that haven’t (due to protocol violations, mislabeling, not having been run, etc.) In other words, it's an archive of every single DICOM that your LORIS instance knows about.
```

Comment:
```suggestion
This module displays all scans that have been uploaded to LORIS. This includes scans that have been inserted, as well as those that were rejected (due to protocol violations, mislabeling, not having been run, etc.) In other words, it's an archive of every single DICOM that your LORIS instance knows about.
```

I'm not sure if 'rejected' is the right word here but I think it's a bit more clear

Reply:
done. kindly re-review @johnsaigle